analyzer/windows/modules/packages/dll.py

```diff
@@ -112,12 +114,24 @@ def start(self, path):
         # To get to this stage, the user has enabled `enable_multi`, and has either specified an ordinal
         # range or requested that we use available exports by ordinal number, up to a limit
         if run_ordinal_range:
+            ret_list = []
             with contextlib.suppress(ValueError, AssertionError):
                 start, end = (int(_.lstrip("#")) for _ in function.replace("..", "-").split("-", 1))
                 assert start < end
-                args = '/c for /l %i in ({start},1,{end}) do @{rundll32} "{path}",#%i {arguments}'.format(**locals())
-                # if there are multiple functions launch them by their ordinal number in a for loop via cmd.exe calling rundll32.exe
-                return self.execute("C:\\Windows\\System32\\cmd.exe", args.strip(), path)
```

Comment:
Changing from this style to the other style does change the execution flow here a bit (originally it was python ->  cmd.exe -> mutliple dlss. Now its python -> multiple dlls. Lets sync with Doron in the morning to make sure it is ok

Reply:
it reviewed by Doron